activestorage/lib/active_storage/attached/model.rb

```diff
@@ -50,8 +50,21 @@ module Attached::Model
       def has_one_attached(name, dependent: :purge_later, service: nil, strict_loading: false)
         validate_service_configuration(name, service)
 
+        instance_eval <<-CODE, __FILE__, __LINE__ + 1
+          # frozen_string_literal: true
```

Comment:
Is this line needed, if the file already has this on line 1?

Reply:
I was also curious about this... it looks like most generated methods have it added (but I'm not sure I understand why yet)

A bunch of them were added in 1ef8c60dfcab2f0594b468088680fe1054d17452

Reply:
I was just following those examples. I'm happy to omit them.